docs/.vuepress/config.js

```diff
@@ -869,17 +869,69 @@ module.exports = {
     },
     '/zh/': {
       lang: 'zh-CN',
-      label: translate('language', 'zh-CN'),
-      title: translate('ethereum', 'zh-CN'),
```

Comment:
We don't want to delete this `title` key.

Reply:
I was using the /se/ in config.js as a reference and this is how it looks: https://github.com/ethereum/ethereum-org-website/blob/3ece6c6fb9fc63bd529dc39448735ba374d48cad/docs/.vuepress/config.js#L669

```
'/se/': {
      lang: 'sv-SE',
      label: translate('language'),
      nav: [
      ...
```
This is how it looks for other v1.1 translations too

Reply:
Also in #816 I see the same way

Reply:
Thanks! You're right - I don't believe this is actually needed.

Reply:
Should I revert the commit I did?